cpp/purify/kernels.cc

```diff
@@ -139,7 +138,7 @@ t_real ft_pswf(const t_real x, const t_real J) {
   const t_real alpha = 1;
   const t_real eta0 = 2 * x;
 
-  return calc_for_pswf(eta0, J, alpha);
+  return calc_for_pswf(eta0, J, alpha) * std::sqrt(5.343);
```

Comment:
I see... `2e`, right? Isn't a better way to keep that value somewhere, like in `types.h` like for `pi`?

Reply:
So this normalisation was done by hand, because I don't know the analytic factor. `2e` looks really close! I will try it out.

Reply:
It looks like `2e` is too far away and would make tests fail :(

Reply:
Oohh! so it's a random number that looks like `2e`?? Ha! Well, write then a small comment so that's understood ;)